### DISCUSSION TITLE
ST7701 MaTouch ESP32-S3 Rotary IPS Display with Touch 2.1“ - can't figure how to run it on 1.4.0

### DISCUSSION THREAD

**Discussion by banoz** [moononournation/Arduino_GFX#383] [Q&A]
Hi, I'm trying to build the example code written for 1.3.1 on 1.4.0 and can't figure how to migrate it. Arduino_ESP32RGBPanel constructor previously accepted CS, SCK, SDA pins of the display. It looks like in 1.4.0 I need to use some Arduino_DataBus, like Arduino_Wire, for that, but I'm a total noob and need an example. Can somebody help please?

**Comment by moononournation** [comment]
You may find some example in PDQgraphicstest.ino
